launcher/ui/widgets/InfoFrame.cpp

```diff
@@ -97,48 +97,60 @@ void InfoFrame::updateWithResource(const Resource& resource)
     setImage();
 }
 
-// https://www.sportskeeda.com/minecraft-wiki/color-codes
-static const QMap<QChar, QString> s_value_to_color = {
-    {'0', "#000000"}, {'1', "#0000AA"}, {'2', "#00AA00"}, {'3', "#00AAAA"}, {'4', "#AA0000"},
-    {'5', "#AA00AA"}, {'6', "#FFAA00"}, {'7', "#AAAAAA"}, {'8', "#555555"}, {'9', "#5555FF"},
-    {'a', "#55FF55"}, {'b', "#55FFFF"}, {'c', "#FF5555"}, {'d', "#FF55FF"}, {'e', "#FFFF55"},
-    {'f', "#FFFFFF"}
-};
-
 QString InfoFrame::renderColorCodes(QString input) {
-    // We have to manually set the colors for use.
-    //
-    // A color is set using §x, with x = a hex number from 0 to f.
-    //
-    // We traverse the description and, when one of those is found, we create
-    // a span element with that color set.
-    //
-    // TODO: Make the same logic for font formatting too.
-    // TODO: Wrap links inside <a> tags
+	// We have to manually set the colors for use.
+	//
+	// A color is set using §x, with x = a hex number from 0 to f.
+	//
+	// We traverse the description and, when one of those is found, we create
+	// a span element with that color set.
+	//
+	// TODO: Wrap links inside <a> tags
+
+	// https://www.sportskeeda.com/minecraft-wiki/color-codes
+	const QMap<QChar, QString> color_codes_map = {
+		{'0', "#000000"}, {'1', "#0000AA"}, {'2', "#00AA00"}, {'3', "#00AAAA"}, {'4', "#AA0000"},
+		{'5', "#AA00AA"}, {'6', "#FFAA00"}, {'7', "#AAAAAA"}, {'8', "#555555"}, {'9', "#5555FF"},
+		{'a', "#55FF55"}, {'b', "#55FFFF"}, {'c', "#FF5555"}, {'d', "#FF55FF"}, {'e', "#FFFF55"},
+		{'f', "#FFFFFF"}
+	};
+	const QMap<QChar, QString> formatting_codes_map = {
```

Comment:
```suggestion
	// https://minecraft.fandom.com/wiki/Formatting_codes#Formatting_codes
	const QMap<QChar, QString> formatting_codes_map = {
```

leave a reference to future maintainers! (we :slightly_smiling_face:)